user-guide/query-examples-hacks/cohort-example.md

```diff
@@ -0,0 +1,63 @@
+# Cohort Query Example
+
+Cohorts are a good tool to review retention of users over a defined period of time.
+
+To create your own cohort report, you'll need to define your cohort time frame, population and activeness of users.
+
+Here is an example for a cohort query in PostgreSQL, step by step:
```

Comment:
I would give a sentence explaining WITH statements here and how we use them to make the query modular, as not everyone is familiar with the concept.